COBOL Programming Course #1 - Learning COBOL/COBOL Programming Course #1 - Learning COBOL.md

```diff
@@ -20,1072 +20,7 @@ header-includes:
-
-### Install from Public npm Registry 
-To install a Zowe CLI plug-in from the registry, simply locate the plug-in you wish to install, e.g. `@zowe/cics-for-zowe-cli`, find the distribution tag for the distribution you want to install, e.g. `zowe-v1-lts`, and issue the following command: 
-
-```
-zowe plugins install <name>@<distTag>
-```
-
-For example,
-
-```
-zowe plugins install @zowe/cics-for-zowe-cli@zowe-v1-lts
-```
-
-Multiple plug-ins can be installed in a single command. For example, to install all Zowe CLI plug-ins available from the Zowe organization, you could issue:
-
-```{.bash}
-zowe plugins install @zowe/cics-for-zowe-cli@zowe-v1-lts @zowe/ims-for-zowe-cli@zowe-v1-lts @zowe/mq-for-zowe-cli@zowe-v1-lts @zowe/zos-ftp-for-zowe-cli@zowe-v1-lts @zowe/db2-for-zowe-cli@zowe-v1-lts
-```
-
-Vendor plug-ins on the registry are installed in the same way. For example, to install the CA Endevor plug-in, you would issue
-
-```
-zowe plugins install @broadcom/endevor-for-zowe-cli@zowe-v1-lts
-```
-
-### Install from Bundled Package
-Navigate to [Zowe.org Downloads](https://www.zowe.org/#download) and click the CLI Plugins button to download the package which includes all Zowe CLI plug-ins for the Zowe organization. After accepting the EULA for Zowe, a package named `zowe-cli-plugins-v.r.m.zip` will be downloaded to your machine. Unzip the contents of `zowe-cli-plugins-v.r.m.zip` to a preferred location on your machine. You can select which plug-ins you want to install. The IBM Db2 plug-in requires [additional configuration](https://docs.zowe.org/stable/user-guide/cli-db2plugin.html#installing-from-a-local-package) when installing from a local package. To install all plug-ins you can issue:
-
-```
-zowe plugins install cics-for-zowe-cli.tgz zos-ftp-for-zowe-cli.tgz ims-for-zowe-cli.tgz mq-for-zowe-cli.tgz db2-for-zowe-cli.tgz
-```
-
-For offline installation of vendor plug-ins, please reach out to the specific vendor for details.
-
-## Summary
-In this chapter, we walked through the process of installing the prerequisite software, Node.js, and npm, as well as Zowe CLI and various plug-ins.
-
-\newpage
-
-# Part 2 - Learning COBOL {.unnumbered}
+# Learning COBOL {.unnumbered}
```

Comment:
This unnumbered title can be deleted, since it will be a book by itself.

Reply:
Unnumbered title Deleted.